### PR TITLE
OZ-336: Voiding an OpenMRS observation updates observations dataset in Superset

### PR DESCRIPTION
**Pull request by kdaud**
Ticket → [OZ-336](https://mekomsolutions.atlassian.net/browse/OZ-336?focusedCommentId=41838)

[OZ-336]: https://mekomsolutions.atlassian.net/browse/OZ-336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Description → This PR adds E2E test asserting that voiding an OpenMRS observation updates observations dataset in Superset

### CODE REVIEW

**Comment by kdaud** [comment]
This PR will be ready for review when OZ-483 is resolved.

File: e2e/utils/functions/testBase.ts [hunk 1]
```diff
@@ -308,7 +308,7 @@ export class HomePage {
     await delay(5000);
   }
 
-  async discontinueLabOrder() {
+  async discontinueAnOpenMRSEncounter() {
```
Comment (Ruhanga) [inline e2e/utils/functions/testBase.ts:311]: Doesn't seem logical to discontinue an Encounter. Did you mean to void an Encounter instead? If not, then makes sense to leave the function name as is (`discontinueLabOrder`).

**Review by Ruhanga — COMMENTED** [review]
Generally looks good save for one minor concern.

File: e2e/utils/functions/testBase.ts (hunk 1 above)
Comment (kdaud) [reply e2e/utils/functions/testBase.ts:311]: > Did you mean to void an Encounter instead? 
Sure, it makes sense to rename the function to `voidEncounter()`

> stay consistent in your approach to naming methods.
:+1:

**Review by Ruhanga — APPROVED** [review]

**Pull request merged**